### DISCUSSION TITLE
Snack/dinner experiments

### DISCUSSION THREAD

**Discussion by Tessmore** [Tessmore/tessmore#20] [Food]
> [!IMPORTANT]
> Ingredients listed are for two people

## Tofu bites

**Ingredients:**

> [!NOTE]
> It requires quite a bit of garlic/ginger and chopping it is a lot of work. So you can get things done a lot faster if you use pre-peeled or pastes instead. You can add a hotsauce for extra spice and one pepper for decoration

* Tofu (`350 - 400g`) and (corn) flour
* Butter (`50 - 100g`)
* Garlic (plenty)
* Ginger
* Spring onion
* Peppers
* Soy sauces (light/dark)

**Recipe part 1:**

1. Cut tofu in cube/rectangles and coat them in (corn) flour
2. Put minimal amount of oil in pan and get it hot
3. In batches: "Fry" the tofu gold brown

**Recipe part 2:**

1. Chop garlic, ginger, and peppers (if needed). 
2. Cut spring onion stems in half, then in 3cm parts
3. Put oil in pan, medium heat
4. Throw the spring onion, garlic, ginger, peppers
5. (Wait 1 minute)
6. Add butter
7. Put on medium/low heat
8. (Wait 5 minutes)
9. Add soy sauces (half / one table spoon) 
10. Mix in the tofu so they are covered in the sauce
11. (Wait 1 minute)
12. Serve

## Bloemkool (Cauliflower) bites

**Ingredients:**

* 1 Cauliflower -> Split up into bite sized chunks

**Recipe:**

1. Boil water
13. Put hot water with `cauliflower` in pot (low simmer for 10/15 mins to soften up)
14. Pre-heat oven (220°C)
15. Create "cover sauce": 
    - Half glass (120ml) hot water 
    - Some salt
    - Some fish sauce
    - Some soy sauce
    - Some garlic powder
    - Chili powder / chili oil
    - Add flour and stirr until it's a bit thicker
16. Once `cauliflower` is ready -> Strain water -> Put in the "cover sauce" -> Stirr so it's all covered
17. Place `caulifower` on baking tray
    - Sprinkle with black pepper and chili powder to taste
    - Optional/recommended -> Add cheese on top 
18. Put in oven for 15/20 minutes (until edges start to brown)

![WhatsApp Image 2023-11-06 at 17 45 17](https://github.com/Tessmore/tessmore/assets/1188065/9ef15f6d-bce1-4b08-80b7-f431ea9869b3)

19. Serve (soy sauce / chili oil to dip)

**Comment by lilbroek** [comment]
Very lekker <3
